code_test.go

```diff
@@ -84,23 +84,20 @@ func TestReqGraph_ParseCode(t *testing.T) {
 	expectedTags := []TagMatch{
 		{"enumerateObjects",
 			30,
-			`// This method does stuff also.
-// @llr REQ-TEST-SWL-13
-// @xlr R-1`},
+			`REQ-TEST-SWL-13`},
```

Comment:
Much nicer, we don't really care about the comment, so that was a bit useless 👍🏻